hack/verify-rpm-index.sh

```diff
@@ -2,12 +2,11 @@
 
 # Copyright 2021 Hewlett Packard Enterprise Development LP
 
-PACKAGING_TOOLS_IMAGE="arti.hpc.amslabs.hpecorp.net/internal-docker-stable-local/packaging-tools:0.12.7"
```

Comment:
This var is defined in `vendor/github.hpe.com/hpe/hpc-shastarelm-release/lib/release.sh` (sourced below).